docs/crio.conf.5.md

```diff
@@ -50,7 +50,7 @@ CRI-O reads its storage defaults from the containers-storage.conf(5) file locate
 ## CRIO.API TABLE
 The `crio.api` table contains settings for the kubelet/gRPC interface.
 
-**host_ip**=""
+**host_ips**=[]
```

Comment:
what was the reason for limiting to 2?

Reply:
I think it was the discussion that Kubernetes works only with 1 IPv4 and 1 additional/optional IPv6 right now.